src/main/java/io/github/cottonmc/cotton/gui/widget/WScrollBar.java

```diff
@@ -238,6 +238,14 @@ public WScrollBar setWindow(int window) {
 		return this;
 	}
 
+	public int getScrollingSpeed() {
+		return scrollingSpeed;
+	}
+
+	public void setScrollingSpeed(int scrollingSpeed) {
+		this.scrollingSpeed = scrollingSpeed;
+	}
```

Comment:
These need some javadoc.

It'd also be good to have a note of the default scrolling speed there, for example with `{@value #DEFAULT_SCROLLING_SPEED}` (+ a `DEFAULT_SCROLLING_SPEED` constant).